### PR TITLE
Update: native commands and new wiki parser

### PR DESCRIPTION
**Pull request by simon84**
I updated the wiki parser. Now the output is cleaner and some errors disappeared from `language-sqf-native-commands.json`. Additional I changed the priority, which should deliver better results in autocomplete-plus.

### CODE REVIEW

**Comment by simon84** [comment]
Please wait with merging. I found an error. I will commit later.

**Comment by simon84** [comment]
should be ok now

**Comment by jonpas** [comment]
We'll look at it soon.

Could you look into #18 by any chance?

File: settings/language-sqf.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  ".source.sqf, .source.cpp, .source.hpp, .source.ext": {
+  ".source.sqf, .source.arma.cfg": {
```
Comment (jonpas) [inline settings/language-sqf.json:2]: I am not sure I like this, you need to have "Arma Config Format" selected for that to work, by default those files have "C++" selected and it's just an unnecessary step to choose it. I'd leave it as it was.

**Comment by jonpas** [comment]
Don't worry about the merge conflicts, just check the comment. :)

File: settings/language-sqf.json (hunk 1 above)
Comment (jonpas) [inline settings/language-sqf.json:2]: I've added both so it now supports "Arma Config Format" and "C++".

**Comment by jonpas** [comment]
Came up with a better idea, check https://github.com/acemod/language-arma-atom/pull/26#discussion_r38880096

Thanks, great work, now just BIS functions left! :+1:

**Pull request merged**